<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MIPSWindowClass</class>
 <widget class="QMainWindow" name="MIPSWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1259</width>
    <height>677</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MIPS Simulator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="gCompilerLoader">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>251</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Compilation + Load Unit</string>
    </property>
    <widget class="QTextEdit" name="tEditor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="bLoadSrc">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;Open Source</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bAssemble">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;Assemble + Load</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bLoadConfig">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>170</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;Load Config</string>
     </property>
    </widget>
    <widget class="QLabel" name="lLoadAt">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>200</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Load at</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tLoad">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>200</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>00000000</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gMemory">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>241</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Memory</string>
    </property>
    <widget class="QLabel" name="lStartLoc">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>46</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Start at</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tLoc">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>00000000</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bShow">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;Show</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="tMemory">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>221</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="bSave_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>S&amp;ave</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gRegister">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>261</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Registers</string>
    </property>
    <widget class="QLineEdit" name="tLMD">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>200</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cFPR">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="tPC">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>00000000</string>
     </property>
    </widget>
    <widget class="QLabel" name="lFPR">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>20</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>FPR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bExec">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;Execute</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tIR">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lLMD">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>200</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>LMD</string>
     </property>
    </widget>
    <widget class="QLabel" name="lIR">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>IR</string>
     </property>
    </widget>
    <widget class="QLabel" name="lPC">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>PC</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tFPR">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lGPR">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>GPR</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tOp">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cGPR">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lOp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Op</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tGPR">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="tImm">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lImm">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Imm</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tB">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>110</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lB">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>B</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tA">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lA">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>A</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tALU">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lALU">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ALU</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bSaveReg">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Sa&amp;ve</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bStep">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;Step</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gResult">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>10</y>
      <width>321</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Results</string>
    </property>
    <widget class="QTextEdit" name="tResults">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>301</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gName">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>20</y>
      <width>120</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Sandipan Dey&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;CSEE Department&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;611 - Project&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;UMBC&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>3</width>
       <height>61</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1259</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
